test/piezo.js

```diff
@@ -276,18 +276,32 @@ exports["Piezo"] = {
   },
 
   note: function(test) {
-    test.expect(4);
+    test.expect(9);
 
-    // note delegates to tone;
+    // note delegates to frequncy, which delegates to tone;
```

Comment:
`frequency` is misspelled but this is in an incredibly nitpicky comment. Only change it if you have other reason to edit the test file :)